packages/remark-lint-code-block-style/index.js

```diff
@@ -143,9 +143,19 @@ const remarkLintCodeBlockStyle = lintRule(
     url: 'https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-code-block-style#readme'
   },
   /** @type {import('unified-lint-rule').Rule<Root, Options>} */
-  (tree, file, option = 'consistent') => {
+  function (tree, file, option) {
     const value = String(file)
 
+    if (!option) {
+      const {settings} = this.data()
+      option =
+        !settings || settings.fences === undefined
+          ? 'consistent'
+          : settings.fences
+          ? 'fenced'
+          : 'indented'
```

Comment:
```suggestion
    const fences = (this.data().settings || {}).fences
    option =
      typeof fences === 'boolean'
        ? fences
          ? 'fenced'
          : 'indented'
        : 'consistent'
```

Can you instead:
- Use `(this.data().settings || {}).xxx` to get a prop? In the future we can use `?`, but for now the `|| {}` lets us do the same
- Try and prefer the “xxx is defined” as the earlier branches (so in this case `fences` is a boolean), and leave the `consistent` for the latter branch?

Reply:
Same for the other files :)

Reply:
- [x] I've replaced `{settings} = this.data()` -> `settings = this.data().settings || {}` and made the `'consistent'`/default branches the last.